### DISCUSSION TITLE
MetricGAN+ training

### DISCUSSION THREAD

**Discussion by FTanghu** [speechbrain/speechbrain#1663] [Q&A]
hi，
I train the model MetricGAN+ in speechbrain/recipes/Voicebank/enhance/MetricGAN,without modifying the train.yaml，I found that PESQ scores improved very slowly, compared with the author's experimental log on Google drive.The author's experiment got valid PESQ: 2.51 when it was 70 epoch, but my experiment needed more than 200 epochs to reach this score. Is there something wrong with my training? Can I get PESQ: 3.15 at the end?
I find it takes a long time to train MetricGAN+, more than 3 days on a V100 GPU? Does it support multiple GPU training? If yes, which parameters do I need to modify to correct training? 
Appreciate for your reply.

**Comment by guocj97** [comment]
Hi, I am also studying speech enhancement now, and I also met your similar problem when I reappeared MetricGAN+. I think we can communicate with each other. My email address is on my Github homepage.

**Comment by SitalLiang** [comment]
Hello, I also use V100 GPU. After repeated training, I basically replicated the results of the paper, reaching 3.14. In the case of not using the pre-training model, it does need more epochs to achieve the results, you can choose to use the pre-training model published by the author to reduce the training time.

**Comment by FTanghu** [comment]
> Hello, I also use V100 GPU. After repeated training, I basically replicated the results of the paper, reaching 3.14. In the case of not using the pre-training model, it does need more epochs to achieve the results, you can choose to use the pre-training model published by the author to reduce the training time.

Excuse me, can you finally reach 3.14 without using the pre-training model? Did you modify the parameters in train.yaml? I only got 3.12 in the end

**Comment by SitalLiang** [comment]
I have not changed any parameters, and it is acceptable to have some difference in the training results .




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年4月9日(星期六) 中午1:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [speechbrain/speechbrain] MetricGAN+ training (Issue #1345)





  
你好，我也用V100 GPU。  经过反复训练，我基本复制了论文的结果，达到了3.14。  在不使用预训练模型的情况下，确实需要更多的epoch才能达到效果，可以选择使用作者发布的预训练模型来减少训练时间。 
  
请问，不使用预训练模型最终能达到3.14吗？  你修改train.yaml中的参数了吗？  我最后只得到了3.12 
 
— 
直接回复此邮件， 在 GitHub 上查看 ，或 退订 。 
您收到此消息是因为您发表了评论。 消息 ID： <speechbrain/speechbrain/issues/1345/1093715552 @ github 。 com&gt;
